research/object_detection/dataset_tools/create_oid_tf_record.py

```diff
@@ -42,6 +42,11 @@
 from object_detection.dataset_tools import oid_tfrecord_creation
 from object_detection.utils import label_map_util
 
+try:
+    long        # Python 2
+except NameError:
+    long = int  # Python 3
+
```

Comment:
Could we solve this by just using `int` instead of `long` in the code below? I checked [the documentation for `int`](https://docs.python.org/2/library/functions.html#int), and it states, "If the argument is outside the integer range, the function returns a long object instead."

Reply:
Very nice!  You learn something new every day.  I will make it so.